packages/braintree-integration/src/index.ts

```diff
@@ -25,6 +25,6 @@ export { WithBraintreeLocalMethodsPaymentInitializeOptions } from './braintree-l
 /**
  * Braintree AXO strategies
  */
-export { default as createBraintreeAcceleratedCheckoutCustomerStrategy } from './braintree-accelerated-checkout/create-braintree-accelerated-checkout-customer-strategy';
+export { default as createBraintreeFastlaneCustomerStrategy } from './braintree-fastlane/create-braintree-fastlane-customer-strategy';
```

Comment:
Don't we need to export `WithBraintreeFastlaneCustomerInitializeOptions` as well?